backend/tracim_backend/lib/utils/authentication.py

```diff
@@ -177,15 +199,29 @@ def _acs(self, request: TracimRequest) -> Response:
         except ResponseLifetimeExceed as e:
             return Response(e.__str__())
         identity = authn_response.get_identity()
-        if identity is None or "UserID" not in identity:
+        if identity is None:
             return HTTPBadRequest()
+        idp_name = request.POST.get("RelayState")
+
+        formatted_attributes = {}
+        for key, value in self.saml_idp_config[idp_name]["attribute_map"].items():
+            placeholders = re.findall(r"\${(.*?)}", value)
```

Comment:
/!\ regex: you must explain what pattern you expect to match. Give some examples working (and some not working if there are)

Reply:
note: probably the right way to explain the regex is a comment before the for loop explaining that you are parsing configuration in order to match properties with complex ones. Give somes examples so the developer reading your code immediately understand what is the goal.

Reply:
added details in comment